components/booking/ActiveTimeSlot.tsx

```diff
@@ -0,0 +1,66 @@
+import { View, StyleSheet } from "react-native";
+
+import Colors from "../../constants/color";
+import BodyText from "../text/BodyText";
+import HeaderText from "../text/HeaderText";
+import SubHeaderText from "../text/SubHeaderText";
+
+interface activeTimeSlotProps {
+  licensePlate: string,
+  space: string,
+  timeRemaining: string
+}
+
+const ActiveTimeSlot: React.FC<activeTimeSlotProps> = ({ licensePlate, space, timeRemaining }) => {
+  return (
+    <View style={styles.wrapper}>
+      <View style={styles.topContainer}>
+        <HeaderText
+          text={licensePlate}
+          textStyle={{ color: Colors.black, fontWeight: "bold" }}
```

Comment:
No need to specify bold here since it won't do anything

Reply:
done